visualization/app/codeCharta/util/customConfigBuilder.ts

```diff
@@ -51,7 +53,6 @@ function initializeAppSettings(target: CustomConfig) {
 		},
 		amountOfEdgePreviews: 0,
 		amountOfTopLabels: 0,
-		camera: undefined,
```

Comment:
Same goes here. We should check that this won't produce any errors.

Reply:
it passes my legacy tests :)